test/attr.test.js

```diff
@@ -30,6 +31,22 @@ describe('#attr', () => {
       expect(wrapper.find('span')).to.not.have.attr('disabled')
     })
 
+    it('passes when attribute exists without a value', (wrapper) => {
+      expect(wrapper.find('article')).to.have.attr('itemscope')
+    })
+
+    it('passes when attribute exists with a falsey (but not false/null/undefined) value', (wrapper) => {
+      expect(wrapper.find('article')).to.have.attr('draggable')
+    })
+
+    it('passes when attribute exists but has value `false`', (wrapper) => {
+      expect(wrapper.find('article')).to.not.have.attr('hidden')
```

Comment:
Its all more complicated than these tests indicate, because each attribute has potential subtle differences. For example `hidden`, in React, behaves via boolean values (see [their code](https://github.com/facebook/react/blob/4338c8db4b03cb1ba5f2e229b1baa5a848fe65ab/src/renderers/dom/shared/HTMLDOMPropertyConfig.js#L79)) so `true` will render `hidden` where `false` wont. But for something like [`spellCheck`](https://github.com/facebook/react/blob/4338c8db4b03cb1ba5f2e229b1baa5a848fe65ab/src/renderers/dom/shared/HTMLDOMPropertyConfig.js#L140), passing `false` will render `spellcheck="false"` and true `spellcheck="true"`. There's lots of esoteric values for a [range of attributes](https://github.com/facebook/react/blob/4338c8db4b03cb1ba5f2e229b1baa5a848fe65ab/src/renderers/dom/shared/HTMLDOMPropertyConfig.js#L43-L53) - we could test them a subset of them individually if you'd be happier with that?